### PR TITLE
add disable-pop-up flag + add logout command

### PR DESCRIPTION
**Pull request by mattkim**
## What was changed

add disable-pop-up flag + add logout command

## Why?

add logout command to log out of current session

add disable-pop-up flag for login/logout to allow user to pick which browser to open url

## Checklist

1. Closes 

2. How was this tested:

unit tests

3. Any docs updates needed?

### CODE REVIEW

File: app/logout.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+)
+
+func NewLogoutCommand(c *LoginClient) (CommandOut, error) {
+	return CommandOut{Command: &cli.Command{
+		Name:    "logout",
+		Usage:   "Logout current user",
+		Aliases: []string{"lo"},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "domain",
```
Comment (feihuang) [inline app/logout.go:15]: can we reuse the same flag in login.go?

File: app/logout.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+				Value:    "login.tmprl.cloud",
+				Aliases:  []string{"d"},
+				Required: false,
+				Hidden:   true,
+			},
+			&cli.BoolFlag{
```
Comment (feihuang) [inline app/logout.go:21]: same as above

File: app/logout.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+				Name:     "disable-pop-up",
+				Usage:    "disable browser pop-up",
+				Required: false,
+			},
+		},
+		Action: func(ctx *cli.Context) error {
+			logoutURL := fmt.Sprintf("https://%s/v2/logout", ctx.String("domain"))
+			fmt.Printf("Logout via this url: %s\n", logoutURL)
+			if !ctx.Bool("disable-pop-up") {
```
Comment (feihuang) [inline app/logout.go:30]: does it ever make sense to pop up?

**Review by feihuang — APPROVED** [review]

File: app/logout.go (hunk 1 above)
Comment (mattkim) [reply app/logout.go:15]: done

File: app/logout.go (hunk 2 above)
Comment (mattkim) [reply app/logout.go:21]: done

File: app/logout.go (hunk 3 above)
Comment (mattkim) [reply app/logout.go:30]: Yes, i believe pop-up is a good default behavior so you don't have to click on a link

**Pull request merged**